scss/_orange-footer.scss

```diff
@@ -0,0 +1,209 @@
+.footer-mailing {
+  padding-top: $footer-orange-mailing-padding-top;
+  padding-bottom: $footer-orange-mailing-padding-bottom;
+
+  @include media-breakpoint-up(md) {
+    padding-bottom: $footer-orange-mailing-padding-bottom-md;
+  }
+
+  h3 {
+    margin-bottom: $footer-orange-mailing-title-margin-bottom;
+    font-size: $footer-orange-font-size-md;
+    line-height: $footer-orange-line-height-md;
+  }
+
+  form {
+    display: flex;
+    gap: $footer-orange-gap-sm;
+
+    @include media-breakpoint-up(md) {
+      gap: $footer-orange-gap-md;
+    }
+
+    > input {
+      min-width: $footer-orange-mailing-input-min-width;
+      max-width: $footer-orange-mailing-input-max-width;
+    }
+  }
+}
+
+.footer-social {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding-top: $footer-orange-social-padding-top;
+  padding-bottom: $footer-orange-social-padding-bottom;
+
+  @include media-breakpoint-up(md) {
+    padding-top: $footer-orange-social-padding-top-md;
+  }
+
+  > h3 {
+    width: 100%;
+    margin: $footer-orange-social-title-margin;
+    font-size: $footer-orange-font-size-sm;
+    font-weight: $font-weight-bold;
+    line-height: $footer-orange-line-height-sm;
+    letter-spacing: $footer-orange-letter-spacing;
+
+    @include media-breakpoint-up(md) {
+      width: auto;
+      margin: $footer-orange-social-title-margin-md;
+      font-size: $footer-orange-font-size-md;
+      line-height: $footer-orange-line-height-md;
+    }
+  }
+
+  ul {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: $footer-orange-gap-sm;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+  }
+}
+
+.footer-nav {
+  h3 {
+    padding-top: $footer-orange-nav-margin-top;
+    font-size: $footer-orange-font-size-md;
+    line-height: $footer-orange-line-height-md;
+  }
+
+  .navbar-nav {
+    display: flex;
+    flex-direction: column;
+    gap: $footer-orange-gap;
+    align-items: flex-start;
+    padding: $footer-orange-nav-list-padding;
+
+    @include media-breakpoint-up(lg) {
+      padding: $footer-orange-nav-list-padding-md;
+    }
+
+    .nav-link {
+      padding: 0;
+      outline-offset: map-get($spacers, 1);
+    }
+  }
+
+  .accordion {
+    .accordion-header {
+      padding-top: $footer-orange-nav-accordion-header-padding-top;
+      padding-bottom: $footer-orange-nav-accordion-header-padding-bottom;
+      border: 0;
+
+      .accordion-button {
+        width: 100%;
+        padding: 0;
+        margin: 0;
+        font-size: $footer-orange-font-size-sm;
+        line-height: $footer-orange-line-height-sm;
+        color: $white;
```

Comment:
Maybe there's a need / pre-requisite to have an inversed version of accordions? I think that colors are not linked to the footer himself but more to a bg-dark version of accordions